networking_ccloud/common/config/config_oslo.py

```diff
@@ -23,8 +23,16 @@
                help="Path to yaml config file"),
 ]
 
+cc_fabric_agent_opts = [
+    cfg.BoolOpt('persist_config_loop_enabled', default=True,
+                help="Enable/Disable the loop that tells the switch to persists its config"),
+    cfg.IntOpt('persist_config_loop_interval', default=15 * 60,
+               help="Interval between config persists, if loop is enabled"),
+]
```

Comment:
In Nova, this is usually not handled by another variable, but by setting `persist_config_loop_interval` to a negative number, e.g. [here](https://github.com/sapcc/nova/blob/ba9c8e612e0d0e86010b076077b255e409cdc493/nova/conf/compute.py#L770-L773)

Reply:
I thought about doing it this way, but deemed it "user unfriendly". But if this is the OpenStack default, then I happily oblige :)